packages/desktop-client/src/components/reports/ReportSummary.tsx

```diff
@@ -130,7 +130,7 @@ function ReportSummary({
           }}
         >
           <PrivacyFilter blurIntensity={7}>
-            {integerToCurrency(Math.round(average))}
+            {average && integerToCurrency(Math.round(average))}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏if `average <= 0` - it won't show it. Is that expected? Or did you just try to catch `NaN` value (you can use `isNaN` for that)?